package/_service

```diff
@@ -0,0 +1,14 @@
+<services>
+  <service name="tar_scm">
+    <param name="url">https://github.com/SUSE-Enceladus/public-cloud-info-service.git</param>
+    <param name="scm">git</param>
+    <param name="version">2.0.16</param>
+    <param name="revision">refs/tags/v2.0.16</param>
+    <param name="filename">pint-server</param>
+  </service>
```

Comment:
Note that tar_scm has params that allow it to generate a .changes file entry based upon git commits since the previous version...